### PR TITLE
[v0.3] Update start_e2eaiok_docker related README

### PR DESCRIPTION
**Pull request by zigzagcai**
1. Update start_e2eaiok_docker related README

### CODE REVIEW

File: e2eAIOK/DeNas/README.md [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-```
-$ docker run --shm-size=10g -it --privileged --network host --device=/dev/dri -v ${dataset_path}:/home/vmagent/app/dataset -v ${e2eaiok_codebase}:/home/vmagent/app/e2eaiok -w /home/vmagent/app/ e2eaiok-pytorch120 /bin/bash
-$ conda activate pytorch-1.20.0
+# Environment Setup
+``` bash
+# Setup ENV
+git clone https://github.com/intel/e2eAIOK.git
+cd e2eAIOK
+git submodule update --init --recursive
+python3 scripts/start_e2eaiok_docker.py -b pytorch120 -w ${host0} ${host1} ${host2} ${host3} --proxy ""
```
Comment (xuechendi) [inline e2eAIOK/DeNas/README.md:9]: @zigzagcai , can you also add ssh cmdline as rnnt to tell user what is expected port?

Comment (zigzagcai) [reply e2eAIOK/DeNas/README.md:9]: @xuechendi DeNasREADME updated: DeNas enter docker with port numer

**Comment by zigzagcai** [comment]
Completed and ready for merge.

**Review by xuechendi — APPROVED** [review]
LGTM

**Pull request merged**